indexer/cell_coverer.hpp

```diff
@@ -2,54 +2,58 @@
 
 #include "indexer/cell_id.hpp"
 
+#include "base/buffer_vector.hpp"
+
 #include "std/queue.hpp"
 #include "std/vector.hpp"
 
 // TODO: Move neccessary functions to geometry/covering_utils.hpp and delete this file.
 
+constexpr int SPLIT_RECT_CELLS_COUNT = 512;
```

Comment:
Just move it to the CoverRect() function, and call vector::reserve() in the function, not before call.